### ISSUE TITLE
exec: "com.docker.cli": executable file not found in $PATH

### ISSUE THREAD

**Issue by otavioarc** [ZupIT/ritchie-cli#914]
**What happened**:
I'm trying to run the hello-world demo formula using docker but I'm getting the error `exec: "com.docker.cli": executable file not found in $PATH` when I run `rit demo hello-world --docker` in terminal.

![Screen Shot 2021-04-26 at 00 40 37](https://user-images.githubusercontent.com/27450915/116025948-06ec8a80-a628-11eb-92c2-b311ebd30232.png)

**What you expected to happen**:
Print the message:

```plaintext
Hello World!
My name is Otavio.
I'm excited in creating new formulas using Ritchie.
Today, I want to automate everything.
My secret is I like potato.
```
**How to reproduce it (as minimally and precisely as possible)**:
Execute a fresh install of docker and Ritchie CLI, initialize Ritchie and execute `rit demo hello-world --docker` in terminal.

**Anything else we need to know?**:

**Environment**:
- Ritchie version (use `rit --version`): 2.10.3
- Operating System: MacOS BigSur
- Network plugin / Tool and version (if this is a network-related / tool bug):
- Others:
  - CPU: Apple Silicon M1
  - Docker version: 20.10.5

**Comment by GuillaumeFalourd** [comment]
Hi @otavioarc , thank you for reporting this issue 👍🏼 

This error isn't related to Ritchie, but to Docker for macOS users. I suggest you take a look at [this post on StackOverflow](https://stackoverflow.com/questions/64120020/exec-com-docker-cli-executable-file-not-found-in-path), and update your Docker configurations.

Could you try it and tell us if it worked for you?

We plan to update our documentation with this observation ASAP.

**Comment by otavioarc** [comment]
Hi @GuillaumeFalourd . Thanks for the answer.

I tried the possible solutions reported in the StackOverflow question, but I've had no success.

Probably a downgrade to a version which still have the **Disable Cloud Experience** option and later updating the docker to the most recent version would work, but as I'm using Apple Silicon M1 CPU I can't run versions prior to 3.3.1.

I also tried to fix it through docker config files but it didn't work.

I'll keep my eyes on the issue you mentioned, and update my issue as soon as I found a solution.

Thanks for the support.
